austation/code/modules/power/fuel_rods.dm

```diff
@@ -55,11 +55,31 @@
 	var/expended = FALSE // have we removed the TC already?
 	var/multiplier = 3 // how much do we multiply the inserted TC by?
 
+/obj/item/twohanded/required/fuel_rod/bluespace
+	name = "Bluespace Crystal Fuel Rod"
+	desc = "A heavy-duty low-grade fuel rod which fissiles much slower than standard counterparts. However, it's complex and sophisticated design allows insertion and utilisation of a certain material with special properties, namely bluespace crystals."
+	icon_state = "bluespace"
+	fuel_power = 0.05 // bluespace growth shouldn't be *that* interesting, isn't it?
```

Comment:
This will take ages to deplete

Reply:
Depletion speed doesn't depend on `fuel_power`. The only thing that can modify depletion speed is nitryl at this moment. unless you have unannounced changes in RBMK code (which can be considered as a lack of communication on your side.) Even if it did, it doesn't matter since with current code every non-depleted fuel rod's `fuel_power` gets set to 0.1 anyways. (That part seems to be from sloppy proc overriding, but whatever.)